### ISSUE TITLE
Native GUI Section

### ISSUE THREAD

**Issue by deech** [haskell-lang/haskell-lang#87]
On the `Libraries` page I was thinking that a `Native GUI` section might be helpful since people do seem to ask about that. If that's ok, I'd like to submit a PR to have [FLTKHS](http://hackage.haskell.org/package/fltkhs) added to it. Thoughts?

**Comment by bitemyapp** [comment]
Makes sense to me. Perhaps gtk/gtk3 should be added since they seem pretty popular? Do you know anything about the state of the Qt libraries? Some haven't been recently updated and I'm supposing this is where your expertise lies as I am figuring you did a survey of native GUI libraries before starting on fltkhs.

**Comment by deech** [comment]
Definitely gtk/gtk3, also wxHaskell and HsQml. Not sure about Qt bindings though, they weren't in a good state the last I looked.

**Comment by bitemyapp** [comment]
I'm no authority on the content this site gets, but that seems sufficiently exhaustive for adding a native GUI section. I think those three plus fltk cover what most people would want for native GUI and I wouldn't recommend the native GUI libraries I've tried not in that list.

**Comment by deech** [comment]
To be clear, I haven't tried wxWidgets or gtk in over 4 years. I chose not to go with them because they were too hard to install. And I haven't tried HsQML, period. So I'll just put up a PR for the 3 that I've tried.
